.travis.yml

```diff
@@ -2,9 +2,21 @@ language: python
 sudo: required
 dist: xenial
 python: 3.7.0
+services:
+- docker
+addons:
+  apt:
+    packages:
+      - docker-ce
+before_install:
+- docker swarm init
+- cp ./.env.example ./.env
 install:
 - pip install pipenv
 - pipenv install
+- make dev
+before_script:
+- counter=0 ; while ((counter < 10)) && ! pg_isready --host=$PGHOST --port=$PGPORT --dbname=$PGDATABASE ; do sleep 3; let counter=counter+1; done
```

Comment:
It seems this is a known issue in the travis community... https://github.com/travis-ci/travis-ci/issues/4842

Reply:
Yep, look like some folks there used the same sort of idling loop as a workaround. Nice to feel like this solution is validated. (seems like a problem that Travis should fix though :) )

Reply:
yeah, agreed. We could always switch to Circle CI or another CI service if this causes issues in the future. I've used Circle CI with all my other projects, but thought I'd change things up a bit because I got a free sticker from them...